datasette/utils/__init__.py

```diff
@@ -116,6 +134,27 @@ def compound_keys_after_sql(pks, start_index=0):
     return "({})".format("\n  or\n".join(or_clauses))
 
 
+def compound_sort_sql(sort, start_index=0):
+    #
+    # A modified version of compound_keys_after_sql supporting sorting by multiple columns in any direction
+    #
+    or_clauses = []
+    pks_left = sort[:]
+    while pks_left:
+        and_clauses = []
+        last = pks_left[-1]
+        rest = pks_left[:-1]
+        for i, pk in enumerate(rest):
```

Comment:
if you want to make it more consistent with the original `compund_keys_after_sql`, use list comprehension here.